pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -170,7 +170,6 @@ func checkUserPartOfLibvirtGroup() (bool, error) {
 
 func fixUserPartOfLibvirtGroup() (bool, error) {
 	logging.Debug("Adding current user to the libvirt group")
-	// Add user to libvirt/libvirtd group based on distro
```

Comment:
This only applies to later releases of Ubuntu/debian. We need to confirm when this happened.

Reply:
This happened in 16.10, I thought I added that quote to the commit log, but apparently not. The Debian developer I asked  could not find any traces of a 'libvirtd' group in the package history, so let's assume they always used 'libvirt'